simtools/util/general.py

```diff
@@ -717,3 +743,27 @@ def rotate(rotation_angle, x, y):
         x_trans[step] = x[step] * np.cos(rotation_angle) - y[step] * np.sin(rotation_angle)
         y_trans[step] = x[step] * np.sin(rotation_angle) + y[step] * np.cos(rotation_angle)
     return x_trans, y_trans
+
+
+def get_log_excerpt(log_file):
+    """
+    Get an excerpt from a log file (last 30 lines).
```

Comment:
Although 30 lines are good, do we want to have the 'n_print_last_lines=30' is function parameter?

Reply:
Sure, added it now.